homeassistant/components/nuheat/__init__.py

```diff
@@ -5,68 +5,26 @@
 
 import nuheat
 import requests
-import voluptuous as vol
 
-from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import (
-    CONF_DEVICES,
     CONF_PASSWORD,
     CONF_USERNAME,
     HTTP_BAD_REQUEST,
     HTTP_INTERNAL_SERVER_ERROR,
 )
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryNotReady
-from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
 from .const import CONF_SERIAL_NUMBER, DOMAIN, PLATFORMS
 
 _LOGGER = logging.getLogger(__name__)
 
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We should keep `CONFIG_SCHEMA`, but mark it deprecated.

```py
CONFIG_SCHEMA = cv.deprecated(DOMAIN)
```

This gives the user at least a log/warning that their YAML entry isn't used.

Reply:
done